server/apps/rules/routing_rules.py

```diff
@@ -140,6 +143,10 @@ class RoutingRuleSchemeResource(Resource):
                             },
                             'hour_of_day_to': {
                                 'type': 'string'
+                            },
+                            'time_zone': {
+                                'type': 'string',
+                                'nullable': True
```

Comment:
To not lose too much time on guessing what we should or should not support (:smile:), I'll just specify the 'UTC' as a default  time zone in schema as suggested. It's also easier to break backward compatibility now when the system is not in production yet.